src/EditorFeatures/Text/Extensions.cs

```diff
@@ -25,7 +25,7 @@ public static ITextBuffer GetTextBuffer(this SourceTextContainer textContainer)
             var textBuffer = TryGetTextBuffer(textContainer);
             if (textBuffer == null)
             {
-                throw new ArgumentException(TextEditorResources.TextcontainerIsNotAnSource, "textContainer");
+                throw new ArgumentException(TextEditorResources.TextContainerIsNotASource, "textContainer");
```

Comment:
Consider changing the resource name to something like TextContainerNotFromBuffer.  The one we had was auto-generated based on the first part of it's contents, but something better would be welcome.